.pre-commit-config.yaml

```diff
@@ -56,7 +49,6 @@ repos:
       - id: trailing-whitespace
       - id: check-merge-conflict
       - id: check-ast
-      - id: debug-statements
```

Comment:
removing this seems unrelated. or does ruff do this now?

Reply:
Yes, that's code `T10`